packages/flutter/lib/src/material/dialog.dart

```diff
@@ -318,13 +329,14 @@ class AlertDialog extends StatelessWidget {
   /// will not fit.
   final Widget? content;
 
+  // TODO(werainkhatri): how do i mention that 16dp will be used with M3?
```

Comment:
need a little help with this

Reply:
"Defaults to X if [useMaterial3] is true, Y otherwise."